### PR TITLE
additional material component LESQ-237

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

- Updates Lesson Overview mv to 8, tests files, fixtures, schemas and query
- adds expandable container and renders conditional material component
- Updates types on lesson page, components and avo

## Issue(s)

Fixes # LESQ-237

## N.B. Worksheets only load when logged into google as assets are not public currently

1. Go to https://deploy-preview-1816--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/combined-science-secondary-ks4-foundation-aqa/units/measuring-waves/lessons/lesson-1-in-measuring-waves#additionalMaterial
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
Screenshot 2023-08-24 at 17 41 02

How it should now look:
Screenshot 2023-08-25 at 10 44 06

image

## Checklist

- [x] Added or updated tests where appropriate
- [x] Manually tested across browsers / devices
- [x] Considered impact on accessibility
- [x] Design sign-off
- [x] Approved by product owner

### CODE REVIEW

**Review by jack-skerrett-oak — APPROVED** [review]
It might well need some tweaks but they can wait until later when we have had a chat to content squad about permissions etc

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
Sorry, missed this! I think we should just wrap the text when we have to for the nav buttons. Monika isn't here so lets do it and ask for forgiveness

**Review by jack-skerrett-oak — APPROVED** [review]
"Additional material" nav menu item slightly overlaps at the narrowest widths, we are accepting this for now

File: src/components/Button/ButtonInner.tsx [hunk 1]
```diff
@@ -163,7 +163,7 @@ const ButtonInner: FC<ButtonInnerProps> = (props) => {
         </Flex>
       )}
 
-      <Box $position={"relative"}>
+      <Box $position={"relative"} $minWidth={0}>
```
Comment (kimon-satan) [inline src/components/Button/ButtonInner.tsx:166]: why this ? Is it a workaround for something ?

File: src/components/Button/ButtonLabel.tsx [hunk 2]
```diff
@@ -5,6 +5,9 @@ import typography from "../../styles/utils/typography";
 import { Span } from "../Typography";
 
 const ButtonLabel = styled(Span)`
+  display: inline-block;
```
Comment (kimon-satan) [inline src/components/Button/ButtonLabel.tsx:8]: Again a bit concerned seeing these sorts of unexplained style changes at such a low level. Have you checked for unintended effects on other buttons ?

**Review by kimon-satan — CHANGES_REQUESTED** [review]

File: src/components/Button/ButtonLabel.tsx (hunk 2 above)
Comment (k-huggs) [reply src/components/Button/ButtonLabel.tsx:8]: Paired with @mantagen on this as the buttons weren't behaving as expected when text was long the button / container wouldn't shrink with the resizing of the page, which was causing buttons with long text to overlap with other components. This is to ensure the ellipsis appears as button shrinks and text overflows

**Review by kimon-satan — APPROVED** [review]

**Pull request merged**